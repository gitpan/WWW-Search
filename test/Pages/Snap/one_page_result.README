This set of test-result pages was created on Tue Jun 29 13:23:53 1999
