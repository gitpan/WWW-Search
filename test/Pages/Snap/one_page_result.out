http://www.kather.net/VisualBasicSource/KnowledgeBase/q174804.txt
http://www2.chinabyte.com/staticpages/builder/builder_download/PRVB/builder_download_PRVB_50_25_last_week.html
http://www.multiplexsoftware.com/research/asp.htm
http://www.cyf-kr.edu.pl/ftp/softlib/full.html
http://sunsite.doc.ic.ac.uk/Mirrors/ftp.microsoft.com/Softlib/index.txt
http://rs2.ch.liv.ac.uk/~dlc/Microsoft.txt
